docs/rules/sort-vars.md

```diff
@@ -52,15 +52,14 @@ var c, d, a, e;
 
 ## Options
 
-```
-"sort-vars": [<enabled>, { "ignoreCase": <boolean> }]
-```
+This rule has an object option:
 
-### `ignoreCase`
+* `"ignoreCase": false` (default) checks the case-sensitivity of the variables order
```

Comment:
Can you combine these bullets into one? We usually do one bullet per option name.